Shared/Samples/Update related features/UpdateRelatedFeaturesView.swift

```diff
@@ -0,0 +1,282 @@
+        var parksFeatureTable: ServiceFeatureTable?
+        
+        /// The preserves feature table.
+        var preservesTable: ServiceFeatureTable?
+        
+        /// The feature currently selected by the user.
+        var selectedFeature: ArcGISFeature?
+        
+        /// The feature that is related to the selected feature.
+        var relatedSelectedFeature: ArcGISFeature?
+        
+        /// The location of the callout on the map.
+        var calloutPlacement: CalloutPlacement?
+        
+        /// The current visitor count attribute value.
+        var attributeValue = ""
+        
+        /// The name of the selected park.
+        var parkName = ""
+        
+        /// Visitor options for selection.
+        @ObservationIgnored var visitorOptions = ["0-1,000", "1,000–10,000", "10,000-50,000", "50,000-100,000", "100,000+"]
+        
+        /// The currently selected visitor option.
+        var selectedVisitorValue = "0-1,000"
+        
+        /// Clears selected data and callout.
+        func clearAll() {
+            relatedSelectedFeature = nil
+            attributeValue = ""
+            calloutPlacement = nil
+        }
+        
+        /// Loads feature tables from the Alaska parks feature service
+        /// and adds them as operational layers to the map.
+        ///
+        /// - Throws: An error if the service geodatabase or tables fail to load.
+        func loadFeatures() async throws {
+            let geodatabase = ServiceGeodatabase(url: .alaskaParksFeatureService)
+            try await geodatabase.load()
+            // Load parks layer.
+            parksFeatureTable = geodatabase.table(withLayerID: 1)
+            if let parksFeatureTable {
+                parksFeatureLayer = FeatureLayer(featureTable: parksFeatureTable)
+                map.addOperationalLayer(parksFeatureLayer!)
+            }
+            // Load preserves layer.
+            preservesTable = geodatabase.table(withLayerID: 0)
+            if let preservesTable {
+                let preservesLayer = FeatureLayer(featureTable: preservesTable)
+                map.addOperationalLayer(preservesLayer)
+            }
+        }
+        
+        /// Updates the related preserve feature with the new "Annual Visitors" value
+        /// and applies the changes to the service geodatabase.
+        ///
+        /// - Parameters:
+        ///   - feature: The related `ArcGISFeature` to be updated.
+        ///   - newValue: The new value to assign to the `ANNUAL_VISITORS` attribute.
+        /// - Throws: An error if the feature fails to load, update, or if apply edits fail.
+        func updateRelatedFeature(at mapPoint: Point, newValue: String) async throws {
+            guard let relatedSelectedFeature else { return }
+            try await relatedSelectedFeature.load()
+            relatedSelectedFeature.setAttributeValue(newValue, forKey: .annualVisitorsKey)
+            attributeValue = newValue
```

Comment:
```suggestion
        /// - Parameter value: The value to assign to the `ANNUAL_VISITORS` attribute.
        /// - Throws: An error if the feature fails to load, update, or if apply edits fail.
        func updateRelatedFeature(using value: String) async throws {
            guard let relatedSelectedFeature else { return }
            try await relatedSelectedFeature.load()
            relatedSelectedFeature.setAttributeValue(value, forKey: .annualVisitorsKey)
            attributeValue = value
```
Suggested doc changes since `mapPoint` is not needed. The callsite on line 199 will need to be updated `try await model.updateRelatedFeature(using: newValue)`.